test/CMakeLists.txt

```diff
@@ -467,12 +456,36 @@ macro(define_runtime_multiprecision_test name)
     endif()
 
     target_compile_definitions(${test_name} PRIVATE TEST_DATA_DIR="${CMAKE_CURRENT_SOURCE_DIR}/data/")
-endmacro()
+endmacro(define_runtime_multiprecision_test)
+
+macro(define_modular_cpp_int_test name)
+    define_runtime_multiprecision_test(${name})
+
+    set(test_name "${CURRENT_PROJECT_NAME}_${name}_test")
+    target_compile_definitions(${test_name} PUBLIC -DTEST_CPP_INT)
+    target_link_libraries(${test_name} no_eh_support)
+    add_dependencies(${CURRENT_PROJECT_NAME}_test_suite_modular_cpp_int_tests ${test_name})
+    set_target_properties(${test_name} PROPERTIES CXX_STANDARD 17)
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+        target_compile_options(${test_name} PRIVATE "-fconstexpr-steps=2147483647")
+    elseif (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        target_compile_options(${test_name} PRIVATE "-fconstexpr-ops-limit=4294967295")
+    endif()
```

Comment:
This is already set in `define_runtime_multiprecision_test`

Reply:
good point, thank you